### ISSUE TITLE
Generated malware analysis objects 'configuration_version' attribute is an integer, when the spec defines it as a string

### ISSUE THREAD

**Issue by zrush-mitre** [oasis-open/cti-stix-generator#10]
Here is the definition of the attribute in the generator:

https://github.com/oasis-open/cti-stix-generator/blob/86801770f1decd3bb71499edc63daf0679926864/stix2generator/stix21_registry.json#L1001

The spec defines it as a string here:

https://docs.oasis-open.org/cti/stix/v2.1/cs01/stix-v2.1-cs01.html#_dw67pa20zss5

**Comment by chisholm** [comment]
Maybe I thought this was one of those integers-as-strings situations.  Hmmm.  Ok, guess it should be changed to be a random word or something.  I will fix.

**Comment by zrush-mitre** [comment]
I'd check the other values under that object type too. It seems a few of them may have the same issue.

**Comment by chisholm** [comment]
And those are all versions.  Ok, I think I did it that way because an integer-as-string seemed like a better representation for a "version" type property than a random word.  Most version representations in software are dotted integer tuples, so they are built out of integers.  So I used an integer.  It was simple and made good enough sense at the time.  Random words are going to look weird, but at least they'll have the correct type.  If this is really important, we could make some kind of "semver" semantic which generates dotted integer tuples...
